CHANGELOG.md

```diff
@@ -1,6 +1,102 @@
+* Requirements: `raw_tiles v0.1`.
+
+### Details
+
+* Use pypi for mapbox-vector and edtf packages. See [#219](https://github.com/tilezen/tilequeue/pull/219).
+* Update output props processing for normalized sql. See [#221](https://github.com/tilezen/tilequeue/pull/221).
+* Create source metadata based on query results. See [#222](https://github.com/tilezen/tilequeue/pull/222).
+* Move kind calculation to python. See [#223](https://github.com/tilezen/tilequeue/pull/223).
+* Enable Shapely speedups when they're available. See [#224](https://github.com/tilezen/tilequeue/pull/224).
+* Retry deleting tiles on S3.. See [#225](https://github.com/tilezen/tilequeue/pull/225).
+* Makes queries per-table. See [#227](https://github.com/tilezen/tilequeue/pull/227).
+* Tidy things up for a data fetcher interface.. See [#228](https://github.com/tilezen/tilequeue/pull/228).
+* Fixture-based data source. See [#229](https://github.com/tilezen/tilequeue/pull/229).
+* Remove hard coded list of layers in conversion. See [#231](https://github.com/tilezen/tilequeue/pull/231).
+* Add a tile status command. See [#232](https://github.com/tilezen/tilequeue/pull/232).
+* Move min zoom check after yaml calculation. See [#233](https://github.com/tilezen/tilequeue/pull/233).
+* Support lists of templates per source. See [#234](https://github.com/tilezen/tilequeue/pull/234).
+* Move tilequeue.postgresql -> tilequeue.query.pool. See [#236](https://github.com/tilezen/tilequeue/pull/236).
+* Correct function used for parser. See [#237](https://github.com/tilezen/tilequeue/pull/237).
+* Rename start/stop to be consistent with rest of system. See [#238](https://github.com/tilezen/tilequeue/pull/238).
+* Add tilequeue command to list stuck tiles. See [#239](https://github.com/tilezen/tilequeue/pull/239).
+* Add a command to process a single tile. See [#240](https://github.com/tilezen/tilequeue/pull/240).
+* Support for local fixtures. See [#241](https://github.com/tilezen/tilequeue/pull/241).
+* Add option to pass all relations to fixture data fetcher.. See [#242](https://github.com/tilezen/tilequeue/pull/242).
+* Modify Unit of work on queue. See [#243](https://github.com/tilezen/tilequeue/pull/243).
+* Add RAWR data fetcher. See [#244](https://github.com/tilezen/tilequeue/pull/244).
+* Use enum34 package to provide enumerations.. See [#247](https://github.com/tilezen/tilequeue/pull/247).
+* Add rawr enqueue and process commands. See [#248](https://github.com/tilezen/tilequeue/pull/248).
+* Add logging to rawr commands. See [#249](https://github.com/tilezen/tilequeue/pull/249).
+* Update sample config with rawr options. See [#250](https://github.com/tilezen/tilequeue/pull/250).
+* Update wof process to enqueue to rawr queue. See [#251](https://github.com/tilezen/tilequeue/pull/251).
+* Remove legacy tilequeue intersect command. See [#252](https://github.com/tilezen/tilequeue/pull/252).
```

Comment:
is the command's name `tilequeue-intersect`? If so, add tick marks &/or list the actual name with tick marks.